.tekton/pipeline-service-ckcp-testing.yaml

```diff
@@ -209,23 +220,23 @@ spec:
                 kubectl -n default exec pod/podman -- sh -c "curl --location -o /usr/local/bin/argocd https://github.com/argoproj/argo-cd/releases/download/v2.4.7/argocd-linux-amd64 && chmod +x /usr/local/bin/argocd && argocd version --client"
                 kubectl -n default exec pod/podman -- sh -c "KUBECTL_VERSION=1.24.3 && curl -LO https://storage.googleapis.com/kubernetes-release/release/v\$KUBECTL_VERSION/bin/linux/amd64/kubectl && chmod +x ./kubectl && mv ./kubectl /usr/local/bin && kubectl version --client=true"
                 kubectl -n default exec pod/podman -- sh -c "wget https://github.com/mikefarah/yq/releases/download/v4.25.1/yq_linux_amd64 -O /usr/local/bin/yq && chmod +x /usr/local/bin/yq && yq --version"
-            - name: ckcp-setup
+            - name: prepare-scripts
               image: quay.io/redhat-pipeline-service/kubectl:1.25
               resources:
                 requests:
                   memory: 500Mi
                   cpu: 300m
               script: |
                 #!/usr/bin/env bash
-                ## Produce a ckcp_setup.sh to assist execte ckcp sciprt
+                echo "Produce a ckcp_setup.sh to assist execte script openshift_dev_setup.sh"
```

Comment:
`execte script` -> `with the execution of`

Reply:
fixed